scripts/plugin.js

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+
+/**
+ * Note: The plugin API is experimental and likely to change over time. Don't
+ * use it if you bet on stability.
+ */
+import React from "react";
+import { Route } from "react-router";
+
+
+export function flattenPluginRoutes(
```

Comment:
nit: since it receives a `plugins` arguments (plural), shouldn't it be `flattenPluginsRoutes` ?